hedera-node/docs/system-accounts-operations.md

```diff
@@ -0,0 +1,98 @@
+# System accounts
+
+There are some differences when calling operations on system accounts in Hedera and in Ethereum.
+This doc covers these differences and describes the expected behavior.
+
+### ExtCode operations
+
+`ExtCodeSize`
+
+**Ethereum:** Get size of an account’s code. If there’s no code returns 0.
+
+**Hedera:** For addresses in range 0x0 - 0x3E8 (0.0.0 - 0.0.1000) it returns 0.
+
+`ExtCodeCopy`
+
+**Ethereum:** Copy an account’s code to memory. If a contract doesn’t exist on the address, it copies empty bytes.
+
+**Hedera:** For addresses in range 0x0 - 0x3E8 (0.0.0 - 0.0.1000) it copies empty bytes.
+
+`ExtCodeHash`
+
+**Ethereum:** Hash of the chosen account's code, the empty hash (0xc5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470) if the account has no code, or 0 if the account does not exist or has been destroyed.
+
+**Hedera:** For addresses in range 0x0 - 0x3E8 (0.0.0 - 0.0.1000) it returns the empty hash.
+
+### Balance operation
+
+**Ethereum:** Returns balance or 0 if an account or contract doesn’t exist on the address. This is valid for any address.
+
+**Hedera:**
+1. For addresses is range 0x0 - 0x2EE (0.0.0 - 0.0.750): returns 0 as if account doesn’t exist.
+2. For addresses is range 0x2EF - 0x3E8 (0.0.751 - 0.0.1000): returns balance or 0 if an account or contract doesn’t exist on the address. This behavior is the same for addresses above 0.0.1000.
+
+### SelfDestruct operation
+
+**Ethereum:** success for any address. If the beneficiary is the same as contract address, it burns the eth and destructs the contract.
+
+**Hedera:**
+1. For addresses is range 0x0 - 0x2EE (0.0.0 - 0.0.750): fail with status `INVALID_SOLIDITY_ADDRESS`.
+2. For addresses is range 0x2EF - 0x3E8 (0.0.751 - 0.0.1000):
+   - success, if account exists and has `receiverSigRequired` == false;
+   - success, if account exists and has `receiverSigRequired` == false and account is `sender`;
```

Comment:
should this be `receiverSigRequired` == true?
also, do we have a test confirming this scenario works as expected (if we don't we should plan to add test coverage)?

Reply:
Correct, updated as per the comment suggests.